rest/applications/instance-post-example-1/instance-post-example-1.5.x.cs

```diff
@@ -0,0 +1,22 @@
+// Download the twilio-csharp library from twilio.com/docs/libraries/csharp
+using System;
+using Twilio;
+using Twilio.Rest.Api.V2010.Account;
+
+class Example
+{
+    static void Main(string[] args)
+    {
+        // Find your Account Sid and Auth Token at twilio.com/console
+        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+        const string authToken = "your_auth_token";
+        TwilioClient.Init(accountSid, authToken);
+
+        var voiceUrl = new Uri("http://demo.twilio.com/docs/voice.xml");
+        var smsUrl = new Uri("http://demo.twilio.com/docs/sms.xml");
+
+        ApplicationResource.Update("AP2a0747eba6abf96b7e3c3ff0b4530f6e",
```

Comment:
This is the opposite, we don't need the extra variables since the semantic value is provided by the named params. Would you mind updating this example?